app/queries/facility_analytics_query.rb

```diff
@@ -57,7 +57,7 @@ def follow_up_patients_by_period
                 .where(blood_pressures: { facility: @facility })
                 .select("DISTINCT ON (blood_pressures.patient_id, #{BloodPressure.date_to_period_sql(@period)}) blood_pressures.user_id AS user_id, patients.deleted_at")
                 .select("blood_pressures.recorded_at AS bp_recorded_at")
-                .order('blood_pressures.patient_id', BloodPressure.date_to_period_sql(@period), 'blood_pressures.recorded_at'),
+                .order('blood_pressures.patient_id', Arel.sql(BloodPressure.date_to_period_sql(@period)), 'blood_pressures.recorded_at'),
```

Comment:
Ah, I see. `Arel.sql` assumes its contents are safe, and this is why you added protections against unknown values for `@period` 👍 

Can the whole list of columns here be wrapped in an `Arel.sql`?